spec/table_spec.rb

```diff
@@ -2559,9 +2559,7 @@
              +------------------+------------------+).gsub(/^ +/, "")
         )
       end
-    end
 
-    context "when `max_table_width` is nil" do
```

Comment:
this was an exact duplicate of the context ending two lines above, so i merged these two

Reply:
👍  Good pickup.